staging/src/k8s.io/kubectl/pkg/describe/describe_test.go

```diff
@@ -3453,6 +3454,37 @@ func TestDescribeStorageClass(t *testing.T) {
 	}
 }
 
+func TestDescribeVolumeAttributesClass(t *testing.T) {
+	f := fake.NewSimpleClientset(&storagev1alpha1.VolumeAttributesClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            "foo",
+			ResourceVersion: "4",
+			Annotations: map[string]string{
+				"name": "bar",
+			},
+		},
+		DriverName: "my-driver",
+		Parameters: map[string]string{
+			"param1": "value1",
+			"param2": "value2",
+		},
+	})
+	s := VolumeAttributesClassDescriber{f}
+	out, err := s.Describe("", "foo", DescriberSettings{ShowEvents: true})
```

Comment:
Could you please compare the output as raw string instead of multiple `contains` for better readability like in here https://github.com/kubernetes/kubernetes/blob/513fdb5422932e3234590d03c4b8d417e7544796/staging/src/k8s.io/kubectl/pkg/describe/describe_test.go#L4892-L4899

Reply:
updated.